lib/rules/no-regex-spaces.js

```diff
@@ -13,6 +13,8 @@ const astUtils = require("../util/ast-utils");
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
Although there are readability issues when this rule is violated, I don't think we should assume there is a bug much/most of the time. (I think this should be a "suggestion".)